apps/web-portal/index.mjs

```diff
@@ -1,9 +1,50 @@
-import {WebPortalExpressApp} from './lib'
-import {config} from './config.mjs'
+import {createLogger} from '@zorko-io/util-logger'
+import {WebPortalExpressApp, setLogger} from './lib'
+import {config} from './config'
+
+const logger = createLogger({})
+
+setLogger(logger)
```

Comment:
we no need it, responsibility for the creation of concrete logger instance is only on `createLogger`

Reply:
done